packages/core/src/components/Autocomplete/Autocomplete.example.jsx

```diff
@@ -0,0 +1,64 @@
+import Autocomplete from './Autocomplete';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import TextField from '../TextField/TextField';
+
+ReactDOM.render(
+  <div>
+    <Autocomplete
+      label="Select from the options below:"
+      itemToString={i => {
+        return i ? i.name : '';
+      }}
+      items={[
+        {
+          id: 'kRf6c2fY',
+          name: 'Cook County, IL'
+        },
+        {
+          id: 'lYf5cGfM',
+          name: 'Cook County, MD'
+        },
+        {
+          id: 'mZfKcGf9',
+          name: 'Cook County, TN'
+        }
+      ]}
+      onChange={selectedItem => console.log(selectedItem)}
+    >
+      <TextField
+        hint="This is an autocomplete field. Begin typing to search for relevant information. The number of results will be updated as you type."
```

Comment:
Could we provide a hint to help the user understand what they can type in order to see autocomplete results? Right now you have to look at the code to understand that you should type `Co..`.  

✍️  Hint suggestion:

> Type `c` then use ARROW keys to change options, ENTER key to make a selection, ESC to dismiss.

Reply:
Done. Changed the example hints to suggested text.